app/models/assignment_repo/creator.rb

```diff
@@ -135,6 +135,7 @@ def push_starter_code!(github_repo_id)
       starter_code_repository = GitHubRepository.new(client, starter_code_repo_id)
 
       assignment_repository.get_starter_code_from(starter_code_repository)
+      setup_repository(starter_code_repository, assignment_repository)
```

Comment:
setup_repository is in push_starter_code for now since we're focusing on issues creation from starter code on a per assignment basis. @tarebyte  what do you think?